### PR TITLE
Update maistra GA version

### PR DESCRIPTION
**Pull request by mkralik3**

### CODE REVIEW

**Comment by FilipB** [comment]
Blocked by https://github.com/openshift/release/pull/50251

**Comment by mkralik3** [comment]
do not merge.
it is waiting for https://github.com/maistra/test-infra/pull/391 to check if permissions are correct ( also `tmp` commit needs to be reverted)

**Comment by Sau1506mya** [comment]
Hi @mkralik3 , please run `make update` to update the periodic jobs as well. Otherwise `lgtm`.

**Comment by mkralik3** [comment]
@Sau1506mya Hello. I have run `make update` in my localhost. However, nothing was changed in files. Did it update something remotely?

**Comment by mkralik3** [comment]
@Sau1506mya can I somehow trigger those periodic jobs here to test if my changes don't break anything there? ( it looks like only rehearse jobs were triggered )

**Comment by Sau1506mya** [comment]
> @Sau1506mya can I somehow trigger those periodic jobs here to test if my changes don't break anything there? ( it looks like only rehearse jobs were triggered )

Hi @mkralik3 , Which periodic job you want to re-trigger?
Btw, periodic job can be triggered by `/pj-rehearse job_name1 job_name2 ...` and so on .

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
MTT and kiali int tests passed
There are some flaky tests in Kiali Cypress v1.73 test container ( and some failures ) .. I am going to investigate.

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by Sau1506mya** [comment]
Hi @mkralik3 , Can you also re-trigger the job for other periodic jobs, they failed previously so required them also to be passed?

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
@Sau1506mya  Hello, only `aws-rosa-hypershift` jobs remain failing. However, it looks like a problem with provisioning clusters. So PR is ready to merge.

**Comment by Sau1506mya** [comment]
> @Sau1506mya Hello, only `aws-rosa-hypershift` jobs remain failing. However, it looks like a problem with provisioning clusters. So PR is ready to merge.

Hi @mkralik3 , Is hypershift job not active for servicemesh: `periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift`? 
I mean is the servicemesh job not being in weekly testing for ROSA STS Hypershift Layered Product scenarios?

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
@Sau1506mya  I don't know.
According to history, it is. https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.14-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift 

Let me rerun the job again ( it failed on some cluster related steps )

Btw, I didn't see a history for `hypershift-4.13-lp-rosa` and `hypershift-4.12-lp-rosa` jobs. Can they be removed if they are not running?

**Comment by mkralik3** [comment]
@Sau1506mya   can I remove the old ones then? (4.12 , 4.13, 4.14)

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-aws periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.15-lp-interop-servicemesh-interop-aws

**Comment by mkralik3** [comment]
@Sau1506mya  I have added `periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift` job and it passed. So PR is ready.

What should I do with those old hypershift-rosa jobs which don't work? (4.12-4.14) Should I delete them or disable them? If yes, how?

**Comment by Sau1506mya** [comment]
Can we retest the ROSA Classic job :`periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-rosa-classic-servicemesh-rosa-aws`?

**Comment by mkralik3** [comment]
/pj-rehearse periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-rosa-classic-servicemesh-rosa-aws

**Comment by Sau1506mya** [comment]
So @mkralik3 , we are good to go. All tests are passing in all the platforms including interop, ROSA Classic and Hypershift.

File: ci-operator/jobs/maistra/maistra-test-tool/maistra-maistra-test-tool-main-periodics.yaml [hunk 1]
```diff
@@ -253,6 +253,87 @@ periodics:
...
+    org: maistra
+    repo: maistra-test-tool
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: servicemesh-hypershift-4.15-lp-rosa-hypershift
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.15-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift
```
[the remark this answers is not in the document]
Comment (mkralik3) [reply ci-operator/jobs/maistra/maistra-test-tool/maistra-maistra-test-tool-main-periodics.yaml:273]: @calebevans thx, I will update all jobs with that.

**Comment by Sau1506mya** [comment]
Hi @mkralik3 , the PR is good to merge but for this all checks need to be passed. It's still in the waiting stage. Can you please try to rebase the PR or make some changes such that it will come up in the stack and then may be it will run the checks. I tried with `hold` but, it didn't worked.

**Comment by Sau1506mya** [comment]
> Hi @mkralik3 , the PR is good to merge but for this all checks need to be passed. It's still in the waiting stage. Can you please try to rebase the PR or make some changes such that it will come up in the stack and then may be it will run the checks. I tried with `hold` but, it didn't worked.

You can try with adding the slack config to these jobs:
`periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.13-lp-interop-servicemesh-interop-aws`
`periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.13-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift`
`periodic-ci-maistra-maistra-test-tool-main-servicemesh-hypershift-4.12-lp-rosa-hypershift-servicemesh-aws-rosa-hypershift-412 `
In this way, a change will be made in the PR and hopefully it will run the checks.

**Comment by mkralik3** [comment]
@Sau1506mya  I have rebased it with master branch and added slack reporter config there. Should I run pj-rehearse again? ( no changes related to jobs were done )

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**